_posts/2020-08-04-hhvm-4.69.markdown

```diff
@@ -0,0 +1,22 @@
+---
+title: "HHVM 4.69"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.69 is released! This release marks the end of support for 4.62;
+HHVM 4.63&ndash;4.68 remain supported, as do the 4.32 and 4.56 LTS releases.
+
+# Breaking Changes
+
+- Directly implementing the same interface twice with multiple generics is now
+  banned, due to incorrect over-permissiveness.
+- The `hhvm.check_return_type_hints` runtime option has been removed.
+- It is now a typechecker error to attempt to directly construct an
+  `IntlIterator`.
+- It is now a type error to have duplicate classes, type aliases, or records
```

Comment:
I don't quite understand. It was already an error to have duplicate classes or type aliasses.
What changes here?

Reply:
+1 an example or clarification would be useful

Reply:
Repo-auth only (dfe31524392e29a9bde87188fe3b38850104b32e), updating

Reply:
so a compilation error, not a typechecker error?

Reply:
yep

Reply:
https://github.com/facebook/hhvm/commit/dfe31524392e29a9bde87188fe3b38850104b32e